Revert-tvOS/Sources/CollectionDataSource.swift

```diff
@@ -7,6 +7,7 @@ final class CollectionDataSource<Object: Collectable, Cell: UICollectionViewCell
   typealias CellConfigurator = (HomeCollectionCell, item: HomeItem) -> Void
   
   required init(collection: CollectableCollection<HomeItem>, configureCell: CellConfigurator, cellIdentifier: String) {
+    self.completeCollection = collection
     self.collection = collection
```

Comment:
I'd go with the opposite, use `filteredCollection` and `collection` for these variable names. 

Doesn't matter much, but seem more clear to me 😉